wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -566,10 +629,35 @@ class Proto3WireProtocCompatibilityTests {
         |"oneofInt32" : 0.0
         |}""".trimMargin()
 
+    private val ALL_64_JSON = """
+        |{
+        |  "myInt64": "-9223372036854775808",
+        |  "myUint64": "9223372036854775808",
+        |  "mySint64": "-9223372036854775808",
+        |  "myFixed64": "9223372036854775808",
+        |  "mySfixed64": "-9223372036854775808",
+        |  "repInt64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "repUint64": ["9223372036854775808", "9223372036854775808"],
+        |  "repSint64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "repFixed64": ["9223372036854775808", "9223372036854775808"],
+        |  "repSfixed64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "packInt64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "packUint64": ["9223372036854775808", "9223372036854775808"],
+        |  "packSint64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "packFixed64": ["9223372036854775808", "9223372036854775808"],
+        |  "packSfixed64": ["-9223372036854775808", "-9223372036854775808"],
+        |  "oneofInt64": "-9223372036854775808",
+        |  "mapInt64Sfixed64": {
+        |    "-9223372036854775808": "-9223372036854775808"
+        |  }
+        |}""".trimMargin()
+
     private const val IDENTITY_ALL_TYPES_JSON = "{}"
 
     private fun <T : kotlin.Any> list(t: T): List<T> {
       return listOf(t, t)
     }
+
+    private const val BIG_POSITIVE_NUMBER = -9223372036854775807L - 1L
```

Comment:
Isn't this `Long.MIN_VALUE`? Maybe we could just inline that rather than have this (somewhat misleading) named constant.

Reply:
Thank you. Fixed.